mesonbuild/environment.py

```diff
@@ -1778,8 +1778,8 @@ def detect_rust_compiler(self, for_machine: MachineChoice) -> RustCompiler:
                     mlog.warning(
                         'Please do not put -C linker= in your compiler '
                         'command, set rust_ld=command in your cross file '
-                        'or use the RUST_LD environment variable. meson '
-                        'will override your seletion otherwise.')
+                        'or use the RUST_LD environment variable, otherwise meson '
+                        'will override your selection.')
```

Comment:
Typo fix noted.

But the sentence is longer and more unwieldy than it used to be.

Reply:
Thanks for taking a look @eli-schwartz,

Is the following better?
```
                        'or use the RUST_LD environment variable, otherwise'
                        'meson will override your selection.')
```

Reply:
As the person who wrote the first version with typo, I think that your version reads better.

Reply:
I agree the rewording looks better with the proposed change. I'm wondering whether the punctuation could still use some tweaking...

Reply:
This MR already makes things better, so merging. Any further tweaks can be done in subsequent MRs.